app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -755,8 +755,13 @@ class BrowserFragment : WebFragment(), LifecycleObserver, View.OnClickListener,
             Settings.getInstance(requireContext()).shouldUseBiometrics() &&
             Biometrics.hasFingerprintHardware(requireContext())
         ) {
+            if (biometricController == null) {
+                biometricController = BiometricAuthenticationHandler(context!!)
```

Comment:
The code that instantiates this object in `onCreate` does a few additional checks that look important.  Maybe move that code here (so it gets run when foregrounding the app) instead of duplicating it.

Reply:
My mistake, the code in `onCreate` actually has fewer checks.  Do you know any reason why this class would need to be initialized early?  If not, probably worth pulling that initialization out of `onCreate`.